test/FunctionalTests/Tests/DataOData/Tests/OData.Reader.Tests/JsonLight/FeedReaderJsonLightTests.cs

```diff
@@ -241,18 +241,18 @@ public void FeedReaderErrorTest()
                     PayloadEdmModel = model,
                     ExpectedException = ODataExpectedExceptions.ODataException("ODataJsonLightResourceDeserializer_CannotReadResourceSetContentStart", "StartObject")
                 },
-                new PayloadReaderTestDescriptor(this.Settings)
-                {
-                    DebugDescription = "Feed with primitive item in feed property value.",
-                    PayloadElement = PayloadBuilder.EntitySet()
-                        .JsonRepresentation("{" +
-                            "\"" + JsonLightConstants.ODataPropertyAnnotationSeparator + JsonLightConstants.ODataContextAnnotationName + "\":\"http://odata.org/test/$metadata#TestModel.DefaultContainer.Cities\"," +
-                            "\"" + JsonLightConstants.ODataValuePropertyName + "\":[ 1 ]" +
-                            "}")
-                        .ExpectedEntityType(cityType, citiesEntitySet),
-                    PayloadEdmModel = model,
-                    ExpectedException = ODataExpectedExceptions.ODataException("ODataJsonLightResourceDeserializer_InvalidNodeTypeForItemsInResourceSet", "PrimitiveValue")
-                },
+                //new PayloadReaderTestDescriptor(this.Settings)
+                //{
+                //    DebugDescription = "Feed with primitive item in feed property value.",
+                //    PayloadElement = PayloadBuilder.EntitySet()
+                //        .JsonRepresentation("{" +
+                //            "\"" + JsonLightConstants.ODataPropertyAnnotationSeparator + JsonLightConstants.ODataContextAnnotationName + "\":\"http://odata.org/test/$metadata#TestModel.DefaultContainer.Cities\"," +
+                //            "\"" + JsonLightConstants.ODataValuePropertyName + "\":[ 1 ]" +
+                //            "}")
+                //        .ExpectedEntityType(cityType, citiesEntitySet),
+                //    PayloadEdmModel = model,
+                //    ExpectedException = ODataExpectedExceptions.ODataException("ODataJsonLightResourceDeserializer_InvalidNodeTypeForItemsInResourceSet", "PrimitiveValue")
+                //},

```

Comment:
It's invalid test or unfix test case?

Reply:
The test is making sure that there is no in-line stream value, which is the whole point of the PR to enable.  So, it's invalid.  I left it here to consider a unit test for validating that it *is* valid, but there are already plenty of tests that confirm that validity, and it's really hard for me to modify these tests because I can't run them locally (I get a System.IO.FileLoadException trying to load the Microsoft.OData.Core assembly because the manifest doesn't match the assembly reference). 

---
In reply to: [259089922](https://github.com/OData/odata.net/pull/1204#discussion_r259089922) [](ancestors = 259089922)

Reply:
I will open an issue to fix the tests.

---
In reply to: [266957966](https://github.com/OData/odata.net/pull/1204#discussion_r266957966) [](ancestors = 266957966,259089922)